lib/numbers.md

```diff
@@ -5,20 +5,49 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+Fixnum : An integer (a whole number with no decimal point).
+Float : A number with a decimal. eg: 3.14
+
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+.abs : Absolute value (numerical value, regardless of whether it is positive or negative.
+% : Modulo, or the remainder of a division operation.
+** : Exponent
+/ : Division
++ : Addition
+- : Subtraction
+* : Multiplication
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+Used with a number, the '+' operation will perform additive math. Used with a String, the '+' operation will merely place the strings next to one another.
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+No, you must convert the string to an integer before performing math operations on it. 
+Use:
+'20'.to_i
```

Comment:
Converting 20 to an integer and dividing by 3 will only yield a whole number (6). 
Using '20'.to_f Will convert our screen into a number with a decimal point, making the expression 20.0 / 3 = 6.66666666666667

We can also, using the original '20'.to_i ,
Just adding a decimal point to our 3. 

(As long as at least one of our two numbers we are working with is a float, we will get the full, accurate result)

'20'.to_i / 3.0 = 6.66666666666667




JEREMYSTOLL.ORG 


> On Jan 6, 2016, at 11:58 AM, Sumeet Jain <notifications@github.com> wrote:
> 
> In lib/numbers.md:
> 
> >  
> >  # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
> >  
> > -Replace me with your response...
> > +No, you must convert the string to an integer before performing math operations on it. 
> > +Use:
> > +'20'.to_i
> What if you wanted to divide the string "20" by an odd number–like 3? What problem would using to_i cause, and how would you avoid it?
> 
> —
> Reply to this email directly or view it on GitHub.
>